starters/nuxt/pages/products/_handle.vue

```diff
@@ -127,8 +127,8 @@ export default {
       });
       if (variant) {
         this.addItem({
-          variant,
-          quantity: this.quantity
+          ...variant,
+          quantity: parseInt(this.quantity)
```

Comment:
Why `parseInt`? Looks like `this.quantity` is already an integer:

<img width="600" alt="product page this quantity number" src="https://user-images.githubusercontent.com/5732000/191774238-5fce7289-30b6-41bc-907b-e31b5cdd62d4.png">